pom.xml

```diff
@@ -9,7 +9,7 @@
    "License"); you may not use this file except in compliance
    with the License.  You may obtain a copy of the License at
 
-     http://www.apache.org/licenses/LICENSE-2.0
+     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This has come up in the past and I am not sure it is ok to change a single character in a license header. I think you'll want to check the LEGAL Jira project and/or the legal ML.

Reply:
Thanks for raising. See here https://issues.apache.org/jira/browse/LEGAL-416 and the result: http://www.apache.org/foundation/license-faq.html#Apply-My-Software The sample refers to https too. Is that OK? I don't mind to revert to HTTP if this causes a not permitted license change.

Reply:
Done with LEGAL-536.